README.md

```diff
@@ -108,15 +108,15 @@ chown pe-puppet /etc/puppetlabs/puppet/ssl/satellite/puppet.example.com-puppet-c
 chown pe-puppet /etc/puppetlabs/puppet/ssl/satellite/puppet.example.com-puppet-client.key
 ```
 
-f. In the Satellite UI, go to *Administer -> Settings -> Auth* and set the `restrict_registered_puppetmasters` parameter to `true`. Additionally, add your Puppet master's FQDN to the `trusted_puppetmaster_hosts` array on the same page; for example, `[puppet.example.com]`.
+f. In the Satellite UI, go to *Administer -> Settings -> Auth* and set the `restrict_registered_puppetmasters` parameter to `true`. Additionally, add your Puppet server's FQDN to the `trusted_puppetserver_hosts` array on the same page; for example, `[puppet.example.com]`.
 
-On Satellite 6.2 (and since Foreman 1.8.0) the `restrict_registered_puppetmasters` setting has been renamed to `restrict_registered_smart_proxies` (labelled "Restrict registered capsules"). `trusted_puppetmaster_hosts` has been given the label "Trusted puppetmaster hosts" in the UX. You can see the actual setting names by mousing over the label.
+On Satellite 6.2 (and since Foreman 1.8.0) the `restrict_registered_puppetmasters` setting has been renamed to `restrict_registered_smart_proxies` (labelled "Restrict registered capsules"). `trusted_puppetserver_hosts` has been given the label "Trusted puppetserver hosts" in the UX. You can see the actual setting names by mousing over the label.
```

Comment:
Satellite 6.5 and older are EOL. That means you should use `restrict_registered_smart_proxies` and don't really mention `restrict_registered_puppetmasters` at all.

Reply:
Have cut out the bit about 6.2 and updated it to refer to trusted_hosts